spec/services/support_interface/notifications_export_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::NotificationsExport do
+  describe '#data_for_export' do
+    let!(:application_choice) { create(:application_choice, :awaiting_provider_decision, course_option: course_options.sample) }
+    let!(:decided_application_choice) { create(:application_choice, :with_offer, course_option: course_options.sample) }
+    let!(:rbd_application_choice) { create(:application_choice, :with_rejection_by_default, course_option: course_options.sample) }
+    let(:course_options) { courses.map { |course| create(:course_option, course: course) } }
+    let(:courses) { create_list(:course, 3, :open_on_apply, provider: provider) }
+    let(:provider) { create(:provider) }
+
+    before do
+      create(:provider_user, :with_make_decisions, providers: [provider], send_notifications: false)
+      create(:provider_user, :with_make_decisions, providers: [provider], send_notifications: true)
+      create(:provider_user, providers: [provider], send_notifications: true)
+    end
+
```

Comment:
Should we also add the shared example `it_behaves_like 'a data export'` which tests documentation?

Reply:
Done, took a bit of refactoring! I'm not completely sure if the outputs for most of the rows are integers or strings. I've assumed that they're integers because they're all the result of a `.count`. The behaviour test doesn't seem to mind what typing I put!